tpchgen/tests/integration_tests.rs

```diff
@@ -61,63 +55,55 @@ fn test_generator<T, G, F>(
 
 #[test]
 fn test_nation_sf_0_001() {
-    test_generator(
-        |_| NationGenerator::new().iter(),
-        "data/sf-0.001/nation.tbl.gz",
-        0.001,
-        |nation| {
-            format!(
-                "{}|{}|{}|{}|",
-                nation.n_nationkey, nation.n_name, nation.n_regionkey, nation.n_comment
-            )
-        },
-    );
+    let _sf = 0.001;
```

Comment:
Kept these in just for consistency with other tests that actually use a scale factor.